src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -600,5 +604,32 @@ private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken r
 
             return refKind;
         }
+
+        internal static void AddNullCheckingErrorsToParameter(DiagnosticBag diagnostics, ParameterSymbol parameter, bool isLambdaOrLocalFunction)
+        {
+            if (parameter.IsNullChecked)
+            {
+                if (Binder.GetWellKnownTypeMember(parameter.DeclaringCompilation, WellKnownMember.System_ArgumentNullException__ctorString, out DiagnosticInfo diag) is null)
+                {
+                    diagnostics.Add(diag, parameter.Locations.FirstOrNone());
```

Comment:
>parameter.Locations.FirstOrNone() [](start = 42, length = 33)

Consider extracting a local since this expression is repeated several times in this method.